### DISCUSSION TITLE
camera settings

### DISCUSSION THREAD

**Discussion by barrettepp** [aaronwmorris/indi-allsky#1162] [General]
I have been getting pretty good night images but my day time images are not very good.   They work okay if they sun is out but when if cloudy I get nothing.   Using player one Neptune -c.  

![Screen Shot 2024-02-27 at 5 27 17 PM](https://github.com/aaronwmorris/indi-allsky/assets/156556984/b555eba7-2646-44a3-b308-1040388f1bb1)
![Screen Shot 2024-02-27 at 5 27 48 PM](https://github.com/aaronwmorris/indi-allsky/assets/156556984/d1f28298-9f19-454c-b225-f1bfebfc024e)
![Screen Shot 2024-02-27 at 5 19 55 PM](https://github.com/aaronwmorris/indi-allsky/assets/156556984/217214c2-c95c-421d-b16e-b751e6372552)

**Comment by aaronwmorris** [comment]
In your logs, what does it report as the max value of the data.  It will say something like the following:

```
processing._detectBitDepth() [575]: Image max value: 4095
```

**Reply by barrettepp** [reply]
what numbers should I be looking for.    


processing._detectBitDepth() [571]: Image max value: 33276

2024-02-28T18:40:34.504010-06:00 raspberrypi [INFO] Image-2-2263/MainThread processing._detectBitDepth() [571]: Image max value: 7780

2024-02-28T18:39:34.444758-06:00 raspberrypi [INFO] Image-2-2263/MainThread processing._detectBitDepth() [571]: Image max value: 9528

**Reply by barrettepp** [reply]
![Screen Shot 2024-02-28 at 6 47 23 PM](https://github.com/aaronwmorris/indi-allsky/assets/156556984/813acd17-766c-4a2e-8ae2-c1bf29789255)

**Reply by aaronwmorris** [reply]
The picture are strange.  It is like the camera is performing some sort of HDR process.  I am thinking about this.

**Reply by barrettepp** [reply]
Here are all my camera settings.  These should have been the ones from today.   

![IMG_5800](https://github.com/aaronwmorris/indi-allsky/assets/156556984/52cf8763-2f09-4eef-9fe9-ff598dc234db)
![IMG_5801](https://github.com/aaronwmorris/indi-allsky/assets/156556984/b8a6bc93-b87b-4371-8543-8ed58079833d)
![IMG_5802](https://github.com/aaronwmorris/indi-allsky/assets/156556984/43d89925-8fce-4cfa-b74b-81e3f41e14f1)
![IMG_5803](https://github.com/aaronwmorris/indi-allsky/assets/156556984/36867715-eccb-4a5b-811d-66d08e4c1b4a)
![IMG_5804](https://github.com/aaronwmorris/indi-allsky/assets/156556984/61b382f9-9cc0-4f22-a38c-17a2a8b3ee42)
![IMG_5805](https://github.com/aaronwmorris/indi-allsky/assets/156556984/fc9799a6-a690-4b88-89c3-06fc0c6fc082)
![IMG_5806](https://github.com/aaronwmorris/indi-allsky/assets/156556984/fd33b421-f112-4cfc-b7a7-cc23e09f41c1)
![IMG_5807](https://github.com/aaronwmorris/indi-allsky/assets/156556984/9b0fa632-7d22-4f44-858d-ea6e1850b209)
![IMG_5808](https://github.com/aaronwmorris/indi-allsky/assets/156556984/a6f4f181-8d40-4d6d-9342-889440a1c47f)
![IMG_5809](https://github.com/aaronwmorris/indi-allsky/assets/156556984/14743976-220f-4774-9f70-d1ce18e443fc)
![IMG_5810](https://github.com/aaronwmorris/indi-allsky/assets/156556984/d38d038a-3860-4dc6-ab33-4e4072a84386)
![IMG_5811](https://github.com/aaronwmorris/indi-allsky/assets/156556984/27719f93-890d-4aba-9b2b-76b3d2648ee2)
![IMG_5812](https://github.com/aaronwmorris/indi-allsky/assets/156556984/230022f6-bc3f-4ff2-b24e-451348928626)
![IMG_5813](https://github.com/aaronwmorris/indi-allsky/assets/156556984/f9a159be-59d9-4abb-9246-29fb5cca3f9a)
![IMG_5814](https://github.com/aaronwmorris/indi-allsky/assets/156556984/e6fcf73b-6305-4129-a6b3-6725348d6cc1)
![IMG_5815](https://github.com/aaronwmorris/indi-allsky/assets/156556984/32c64ed3-1f3a-47ba-8be9-460970e0e730)

it’s funny because the night photos are great…
![IMG_5799](https://github.com/aaronwmorris/indi-allsky/assets/156556984/3a3bfcc9-df24-42b6-986e-2a1dc27a4c22)

**Comment by aaronwmorris** [comment]
I am starting to suspect that 0 or 10 for the gain is just too low, even for the day time.  Maybe you need to try gain 50 or even 100 for daytime.

**Comment by wjschmitt** [comment]
It would be nice if someone could post their image settings for the RPi HQ camera.  Currently I am using mostly the default settings and it doesn't look too bad, but I think it could look better.  I'm overwhelmed by all of the esoteric settings.
